samples/client/petstore/typescript-inversify/api/pet.service.ts

```diff
@@ -11,9 +11,9 @@
  */
 /* tslint:disable:no-unused-variable member-ordering */
 
-import { Observable } from "rxjs/Observable";
-import 'rxjs/add/operator/map';
-import 'rxjs/add/operator/toPromise';
+import { Observable } from 'rxjs/Observable';
+
+import { map } from 'rxjs/operators';
```

Comment:
I think `map` should be added to the reserved keywords

Reply:
I used the standard import of map operator of rxjs as suggested in the documentation.

Reply:
yes, but if a user defines an api parameter named `map`, this clashes, so we should add `map` to the list of reserved keywords, similar to this:
https://github.com/OpenAPITools/openapi-generator/blob/50fddc701eabeb3025cc4df25212b9b757caeb40/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java#L64-L69

Reply:
I thought you refered to the reserved key keywords of javascript and I could not understand you. Thanks to the example. I added map to reserved keywords.

Reply:
I've filed https://github.com/OpenAPITools/openapi-generator/pull/3020 to rename "map" to "GlobalImportOperators" so that `map` doesn't need to be a reserved word.